packages/server-wallet/benchmarks/fast-signature.benchmark.ts

```diff
@@ -24,10 +26,18 @@ async function benchmark(): Promise<void> {
   iter.map(() => signState(state, wallet.privateKey));
   console.timeEnd('signState');
 
+  const stateWithHash = addHash(state);
+
   console.time('fastSignState');
-  const result = iter.map(async () => fastSignState(state, wallet.privateKey));
+  const result = iter.map(async () => fastSignState(stateWithHash, wallet.privateKey));
   await Promise.all(result);
   console.timeEnd('fastSignState');
+
+  const nitroState = toNitroState(state);
+
+  console.time('nativeSignState');
+  iter.map(() => nativeSignState(nitroState, wallet.privateKey));
+  console.timeEnd('nativeSignState');
```

Comment:
This might be a more fair comparison.

```suggestion
  console.time('nativeSignState');
  iter.map(() => nativeSignState(toNitroState(state), wallet.privateKey));
  console.timeEnd('nativeSignState');
```